packages/workbox-precaching/src/cleanupOutdatedCaches.ts

```diff
@@ -19,7 +19,8 @@ import './_version.js';
  * @alias workbox.precaching.cleanupOutdatedCaches
  */
 export const cleanupOutdatedCaches = () => {
-  addEventListener('activate', (event: ExtendableEvent) => {
+  // See https://github.com/Microsoft/TypeScript/issues/28357#issuecomment-436484705
```

Comment:
There are a lot of similar changes to this one, due to TypeScript being unhappy about the callback taking a parameter of type `ExtendableEvent`.

Declaring these functions as being of type `EventListener` seems fairly "safe", but perhaps the more official route would be to override the `addEventListener` definition so that it supports a listener function that takes in an `ExtendableEvent` parameter, instead of just an `Event` parameter.